app/components/explore/side-bar.js

```diff
@@ -19,6 +20,10 @@ export default Component.extend({
     return getDateRanges.bind(this)();
   }),
 
+  showFiltersOnMobile: computed('device.isMobile', 'showFilters', function() {
```

Comment:
Use [`computed.and`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/and) instead

Reply:
actually had to change the logic because the filters weren't showing up on desktop.. Please review now? @niranjan94